WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -318,6 +324,10 @@ + (NSArray *)routes
   if (nil != [settings objectForKey:FB_SETTING_KEYBOARD_PREDICTION]) {
     [FBConfiguration setKeyboardPrediction:[[settings objectForKey:FB_SETTING_KEYBOARD_PREDICTION] boolValue]];
   }
+  if (nil != [settings objectForKey:FB_SETTING_FORCE_TURN_ON_SOFTWARE_KEYBOARD] &&
+      [[settings objectForKey:FB_SETTING_FORCE_TURN_ON_SOFTWARE_KEYBOARD] boolValue]) {
+    [FBConfiguration forceTurnOnSoftwareKeyboard];
```

Comment:
I find it a bit counter-intuitive. While other options have possibility to disable them, this one cannot be disabled once set to true (e.g. setting false value has no effect)

Reply:
Yes, I considered the same point... So I named `force`, but it's true your point.

Reply:
mm, probably the only capability should be better for now.